i18n/en.json

```
@@ -351,6 +352,12 @@
   "admin.billing.subscription.privateCloudCard.freeTrial.description": "We love to work with our customers and their needs. Contact sales for subscription, billing or trial-specific questions.",
   "admin.billing.subscription.privateCloudCard.freeTrial.title": "Questions about your trial?",
   "admin.billing.subscription.privateCloudCard.upgradeNow": "Upgrade Now",
+  "admin.billing.subscription.proratedPayment.buttonText": "Return to {team}",
+  "admin.billing.subscription.proratedPayment.substitle": "Thank you for upgrading to Cloud Professional. You will be charged a prorated amount  for your Cloud Starter plan and Cloud Professional plan based on the number of days and number of users.",
+  "admin.billing.subscription.proratedPayment.title": "You are now subscribed to Cloud Professional",
+  "admin.billing.subscription.proratedPayment.tooltipText": "If you upgrade to {selectedProductName} from Cloud Starter mid-month, you will be charged a prorated amount for both plans.",
```

Let's remove the hardcoded product names here in all these texts. Let's have them be open based on what they are upgrading. ie.:
```suggestion
  "admin.billing.subscription.proratedPayment.substitle": "Thank you for upgrading to {selectedProductName}. You will be charged a prorated amount  for your {currentProductName} plan and {selectedProductName} plan based on the number of days and number of users.",
```

---

ok